.eslintrc.json

```diff
@@ -9,6 +9,9 @@
     "plugin:@typescript-eslint/recommended",
     "plugin:@typescript-eslint/eslint-recommended"
   ],
+  "globals": {
+    "process": true
+  },
```

Comment:
This is for `process.cwd` in `build.mjs`, right?

Reply:
also for `process.env.*`